target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-d7800.dts

```diff
@@ -140,22 +169,28 @@
 
 		usb30@0 {
 			status = "ok";
+
+			pinctrl-0 = <&usb0_pwr_en_pin>;
```

Comment:
I've some more fixes in my local tree. If this PR is really finished, I would like to pull it into my staging tree and apply my changes on top. I will ping both of you afterwards to test the outcome.

I will take care of removing patch 802.

I just need someone who says it's really finished now.